ssa/utils.go

```diff
@@ -206,6 +206,16 @@ func IsKustomization(object *unstructured.Unstructured) bool {
 	return false
 }
 
+// IsEncryptedSecret checks if the given object is a Kubernetes Secret encrypted with Mozilla SOPS.
+func IsEncryptedSecret(object *unstructured.Unstructured) bool {
```

Comment:
Would not introduce this in this package, but rather keep it together with SOPS logic in kustomize-controller.

Reply:
(Or in a generic SOPS package, but in any case, I wouldn't blend SSA with other implementation specifics).